fares-data-build-tool/fdbt-site
# Feature/tfn 641 service list and number of stages cookies

# Description

-   This PR moves the SERVICE_LIST_COOKIE and NUMBER_OF_PRODUCTS_COOKIE to sessions.

# Testing instructions

-   Pull down this branch and run the site locally. Run through the pages that use these cookies/attributes and validate that they still work as they should.
-   Also validate that the matching json at the end of the user journeys is still of the correct format for the SERVICE_LIST_COOKIE (the NUMBER_OF_STAGES cookie is purely to handle errors and is not used for anything meaningful). 

# Type of change


# Checklist:
